script.js

```diff
@@ -0,0 +1,21 @@
+const mobileMenu = document.getElementById('mobile-nav');
+
+document.getElementById('hamburger').onclick = function () {
+  mobileMenu.style.display = 'block';
+};
+
+document.querySelector('.close').onclick = function () {
+  mobileMenu.style.display = 'none';
+};
+
+document.querySelector('.header-menu-li-mobile-first').onclick = function () {
+  mobileMenu.style.display = 'none';
+};
+
+document.querySelector('.header-menu-li-mobile-second').onclick = function () {
+  mobileMenu.style.display = 'none';
+};
+
+document.querySelector('.header-menu-li-mobile-third').onclick = function () {
+  mobileMenu.style.display = 'none';
+};
```

Comment:
I think you can make this much simpler by creating a class such as "hide-menu" or similar and then putting it on all these elements that have the same listner. Then do something like this:

```suggestion
document.querySelectorAll('.hide-menu').forEach(element => element.onclick = function () {
  mobileMenu.style.display = 'none';
};
```

Reply:
Thanks, its changed